src/sentry/mediators/sentry_app_components/preparer.py

```diff
@@ -30,8 +30,8 @@ def _prepare_stacktrace_link(self):
 
         query = {"installationId": self.install.uuid}
 
-        if self.project:
-            query["projectSlug"] = self.project.slug
+        if self.project_slug:
```

Comment:
Are slugs stable? Would it be safer to use ID?

Reply:
I don't disagree -- but I didn't want to change the protocol here.  In general I think the way we use slugs is dangerous considering we both treat them as stable and permanent but also share them freely to new orgs and projects.  It's, strictly no worse than existing approach.